### ISSUE TITLE
get_3d series issue

### ISSUE THREAD

**Issue by marosteg** [camrinbraun/tags2etuff#32]
When calling get_3d, with or without the series specified in the function, it yields this error:

Error in if (is.na(series)) stop("get_series() failed. Double-check that this eTUFF file or tag dataset has time series data.") : 
  the condition has length > 1

In this case, the entire series data frame is subjected to is.na with many potentially TRUE/FALSE readings in result, rather than a single result to assess the IF statement. This also didn't used to be an issue.

**Comment by camrinbraun** [comment]
reproduced this error with:
```
metaTypes <- read.csv("~/Downloads/metaTypes.csv",header=T)

dir <- '~/Google Drive/Shared drives/MPG_WHOI/data/albacore/'

albacore <- list.files(dir)
#albacore <- albacore[albacore %like% "172419"] # generate list of unique fishIDs

z = 14 #B2398
fishID <- albacore[z] # specify the individual

data_dir <- paste(dir, fishID, '/cdb/', sep='')

etuff <- read_archival(paste(data_dir,fishID,"_eTUFF.txt",sep=""),metaTypes = metaTypes)

series <- get_3d(etuff, fG=FALSE)

```
**Error in if (is.na(series)) stop("get_series() failed. Double-check that this eTUFF file or tag dataset has time series data.") :  the condition has length > 1**

**Comment by camrinbraun** [comment]
should be fixed with #35
